common/src/main/scala/com/gu/media/youtube/YouTubeVideos.scala

```diff
@@ -131,13 +132,23 @@ trait YouTubeVideos { this: YouTubeAccess with Logging =>
       case Some(video) => {
         findMistakesInDev(video) match {
           case None => {
-            val set = client.thumbnails().set(id, thumbnail).setOnBehalfOfContentOwner(contentOwner)
+            val setOperation = client.thumbnails().set(id, thumbnail).setOnBehalfOfContentOwner(contentOwner)
 
             // If we want some way of monitoring and resuming thumbnail uploads then we can change this to be `false`
-            set.getMediaHttpUploader.setDirectUploadEnabled(true)
+            setOperation.getMediaHttpUploader.setDirectUploadEnabled(true)
 
             YoutubeRequestLogger.logRequest(YoutubeApiType.DataApi, YoutubeRequestType.UpdateVideoThumbnail)
-            set.execute()
+            try{
+              setOperation.execute()
+            }
+            catch {
+              case exception: Throwable =>
+                log.error(
+                  "Failed to update thumbnail",
```

Comment:
Is it worth adding a bit more context to this log line to aid debugging? Add the atom or video ids, maybe?